user/firefox.md

```diff
@@ -27,12 +27,12 @@ It is also possible to specify beta versions; e.g., `50.0b6`.
 
 In addition to specific version numbers, there are 6 special aliases you can use:
 
-- [`latest`](https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US)
-- [`latest-esr`](https://download.mozilla.org/?product=firefox-esr-latest&os=linux64&lang=en-US)
-- [`latest-beta`](https://download.mozilla.org/?product=firefox-beta-latest&os=linux64&lang=en-US)
-- [`latest-dev`](https://download.mozilla.org/?product=firefox-aurora-latest&os=linux64&lang=en-US)
-- [`latest-nightly`](https://download.mozilla.org/?product=firefox-nightly-latest&os=linux64&lang=en-US)
-- [`latest-unsigned`](https://tools.taskcluster.net/index/artifacts/#gecko.v2.mozilla-release.latest.firefox/gecko.v2.mozilla-release.latest.firefox.linux64-add-on-devel/)
+- `latest` [see at mozilla.org](https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US)
```

Comment:
Perhaps something like "archive link"? These return the binary files, so we can't exactly "see" them.